prismic-model/js/parts/specific-structured-text.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+type SingleOrMulti = 'single' | 'multi';
+type Props = {|
+  label: string,
+  singleOrMulti: SingleOrMulti,
+  htmlTypes: string[],
+  placeholder?: string,
+  useAsTitle?: boolean,
+|}
+
+export default function specificStructuredText({
```

Comment:
Could you explain what this is doing that structured text doesn't do?

Reply:
I think I see what this is doing - it's when you don't want to include HTML elements. I remember adding this to all structured texts as almost innevitably they would be asked for. What are the rules for when we would want to be more restrictive vs permissive and could we capture this in cardigan?

If this is still definitely the functionality we'relooking for could we rather extend the funtionality of `structuredText` as it would be a small change there.

Reply:
I was just focused on replicating exactly what was already in Prismic, but I don't think it hurts to use the structuredText method as is, so will do that